### PR TITLE
Ch integrate houndci 160451589

### PR DESCRIPTION
**Pull request by obulaworld**
#### What does this PR do?
-Integrate HoundCI to the project

#### Description of Task to be completed?
- Integration of houndci to the project to also check for conventions.
- Attached HoundCI badge to the readme file.

#### How should this be manually tested?
- Does not need testing.

#### What are the relevant pivotal tracker stories?
[Integrate HoundCI with badge attached to readme](https://www.pivotaltracker.com/story/show/60451589)

### CODE REVIEW

**Review by Mrshinezee — APPROVED** [review]
LGTM

**Review by Victor-Ugwueze — APPROVED** [review]
LGTM

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      }],
+      "require-jsdoc": ["error", {
+          "require": {
+              "FunctionDeclaration": true,
+              "MethodDefinition": true,
+              "ClassDeclaration": true
+          }
+      }]
+    }
+};
```
Comment (KaiserPhemi) [inline .eslintrc.js:33]: #### Nitpick
Insert new line at the end of file

File: .hound.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```
Comment (KaiserPhemi) [inline .hound.yml]: #### Nitpick
Insert new line at the end of file

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]

**Comment by KaiserPhemi** [comment]
#### Critical
@obulaworld 
PT user story link attached in the PR message is broken. Please check the link again.

File: .eslintrc.js (hunk 1 above)
Comment (obulaworld) [reply .eslintrc.js:33]: Added the new line. Thanks for the feedback

File: README.md [hunk 3]
```diff
@@ -1,3 +1,4 @@
+[![protected_by hound](https://img.shields.io/badge/protected%20by-Hound-blue.svg)](https://houndci.com/repos)
 Authors Haven - A Social platform for the creative at heart.
```
Comment (KaiserPhemi) [inline README.md]: #### Nitpick
Can you please insert a line between the badge and the title. This is still not properly formatted.

screen shot 2018-09-25 at 3 11 03 pm

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (obulaworld) [reply README.md]: Okay...on it right away

Comment (obulaworld) [reply README.md]: Just added the new line to the readme file. Thanks for the feedback

**Review by KaiserPhemi — APPROVED** [review]

**Pull request merged**